apps/dc_tools/odc/apps/dc_tools/index_from_tar.py

```diff
@@ -143,6 +123,7 @@ def process_file(filename, index, fps, mode=None, n_failed=0, doc_transform=None
 
         return n_failed
 
+    # TODO: Release SQLAlchemy with context manager
```

Comment:
release when? It's used until the last moment of this function and then deleted, so what's the point?